thoth/glyph/cli.py

```diff
@@ -93,18 +102,66 @@ def generate(output: str) -> None:
     "--message",
     "-m",
     type=str,
-    default="-",
+    required=True,
     help="Commit message to be classified",
 )
 def classify(message: str) -> None:
     """Generate CHANGELOG entries from the current Git project."""
     _LOGGER.info("Hello, glyph!")
-    if message == "-":
-        click.echo("Please enter message")
-    else:
-        classifier = load_model("../../data/model_commits_v2_quant.bin") 
-        label = classifier.predict(message)
-        click.echo("Label : " + str(label[0][0])[8:])
+    _LOGGER.info("Model Path : " + MODEL_PATH)
+    classifier = load_model(MODEL_PATH) 
+    label = classifier.predict(message.lower())
+    click.echo("Label : " + str(label[0][0])[8:])
+
+@cli.command()
+@click.option(
+    "--path",
+    "-p",
+    type=str,
+    required=True,
+    help="Path to Git repository"
+)
+@click.option(
+    "--start",
+    type=str,
+    default="",
+    help="Starting date"
+)
+@click.option(
+    "--end",
+    type=str,
+    default="",
+    help="End date"
+)
+def classifyrepo(path: str, start: str, end: str) -> None:
+    _LOGGER.info("Model Path : " + MODEL_PATH)
+    classifier = load_model(MODEL_PATH)
+
+    start_time = 0
+    end_time = 4119206400
```

Comment:
Are these hardcoded?

Reply:
Yes.
 0 signifies the lowest start_time
4119206400 is just a large number to signify an end date far out in the future
These values help me in keeping the next if statement simple.

Reply:
It's a good practice not to create such magic constants in the code but rather make checks slightly more complicated. Otherwise the code reaches these magic constants (you never know how the code will be used/refactored) and can create issues.

Reply:
I was actually looking to put the max int value for end_time. Something like Integer.MAX_VALUE in Java. But looks like Python3 got rid of max values for ints. But I can use float('inf') instead. Will update the PR.

Reply:
Please do not mix int and float, they are different types.

Reply:
Can I use sys.maxsize? or its multiples for even larger timestamps?

Reply:
Why not use `None` if the upper/lower boundary is not set?

Reply:
Have to add separate condition for checking that. sys.maxsize works fine for keeping the test condition simple and is not any arbitrary constant.